evaluators/embedding/InfiniteNormEvaluator/__init__.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np
+
+from jina.executors.evaluators.embedding import BaseEmbeddingEvaluator, expand_vector
+
+
+class InfiniteNormEvaluator(BaseEmbeddingEvaluator):
+    """A :class:`InfiniteNormEvaluator` evaluates the distance between actual and desired embeddings computing
+    the Infinite Norm between them
+    """
+
+    @property
+    def metric(self):
+        return 'InfiniteNormEvaluator'
+
+    def evaluate(self, actual: 'np.array', desired: 'np.array', *args, **kwargs) -> float:
+        """"
+        :param actual: the embedding of the document (resulting from an Encoder)
+        :param desired: the expected embedding of the document
+        :return the evaluation metric value for the request document
+        """
+        actual = expand_vector(actual)
+        desired = expand_vector(desired)
+        return _infinitenorm(actual, desired)
+
+
+def _get_ones(x, y):
+    return np.ones((x, y))
+
+
+def _ext_A(A):
```

Comment:
remove all the unused methods.

Reply:
Keeping this as it will required to calculated norms on transformed `np.array` A